projects/samples/contests/robocup/controllers/referee/referee.py

```diff
@@ -1740,39 +1744,40 @@ def stop_penalty_shootout():
     kickoff_team_leads = kickoff_team.score >= game.state.teams[0].score and kickoff_team.score >= game.state.teams[1].score
     penalty_shootout_count = game.penalty_shootout_count % 10  # supports both regular and extended shootout kicks
     if (penalty_shootout_count == 6 and diff == 3) or (penalty_shootout_count == 8 and diff == 2):
-        info(message)
         return True  # no need to go further, score is like 3-0 after 6 shootouts or 4-2 after 8 shootouts
     if penalty_shootout_count == 7:
         if diff == 3:  # score is like 4-1
-            info(message)
             return True
         if diff == 2 and not kickoff_team_leads:  # score is like 1-3
-            info(message)
             return True
     elif penalty_shootout_count == 9:
         if diff == 2:  # score is like 5-3
-            info(message)
             return True
         if diff == 1 and not kickoff_team_leads:  # score is like 3-4
-            info(message)
             return True
     return False
 
 
 def next_penalty_shootout():
     game.penalty_shootout_count += 1
-    if not game.penalty_shootout_goal and not game.state.game_state[:6] == "FINISH":
+    if not game.penalty_shootout_goal and not game.state.game_state[:6] == "FINISHED":
```

Comment:
That looks wrong:
```python
game.state.game_state[:6] == "FINISHED"
```
is always false. We should change it to:
```python
game.state.game_state[:8] == "FINISHED"
```

Reply:
You're right, I was surprised by the `FINISH` instead of `FINISHED` but I didn't took into account the `[:6]`... I will fix it.

Reply:
That's solved in ca4bdf0b54277e